ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -53,135 +53,263 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
+    assert.dom(tab('Destinations')).hasText('Destinations', 'Destinations tab renders');
+    assert.dom(overview.createDestination).hasText('Create new destination', 'Toolbar action renders');
   });
 
-  test('it should render landing cta component for community', async function (assert) {
-    this.version.type = 'community';
-    this.version.features = [];
-    this.destinations = [];
+  module('community', function (hooks) {
+    hooks.beforeEach(function () {
+      this.version.type = 'community';
+      this.version.features = [];
+      this.destinations = [];
+      this.activatedFeatures = [];
+    });
 
-    await this.renderComponent();
+    test('it should show an upsell CTA', async function (assert) {
+      await this.renderComponent();
 
-    assert.dom(title).hasText('Secrets Sync Enterprise feature', 'Page title renders');
-    assert.dom(cta.button).doesNotExist('Create first destination button does not render');
+      assert
+        .dom(title)
+        .hasText('Secrets Sync Enterprise feature', 'page title indicates feature is only for Enterprise');
+      assert.dom(cta.button).doesNotExist();
+    });
   });
 
-  test('it should render landing cta component for enterprise without the secrets sync feature', async function (assert) {
-    this.destinations = [];
-    this.version.features = [];
-    await this.renderComponent();
+  module('enterprise', function (hooks) {
+    hooks.beforeEach(function () {
+      this.destinations = [];
+    });
 
-    assert.dom(title).hasText('Secrets Sync Premium feature', 'Page title renders');
-    assert.dom(cta.button).doesNotExist('Create first destination button does not render');
-    assert.dom(cta.summary).exists('CTA renders');
-  });
+    test('it should show an upsell CTA if license does NOT have the secrets sync feature', async function (assert) {
+      this.version.features = [];
+      await this.renderComponent();
 
-  test('it should render header, tabs and toolbar for overview state', async function (assert) {
-    await this.renderComponent();
+      assert
+        .dom(title)
+        .hasText('Secrets Sync Premium feature', 'title indicates feature is only for Premium');
+      assert.dom(cta.button).doesNotExist();
+      assert.dom(cta.summary).exists();
+    });
 
-    assert.dom(title).hasText('Secrets Sync', 'Page title renders');
-    assert.dom(breadcrumb).exists({ count: 1 }, 'Correct number of breadcrumbs render');
-    assert.dom(breadcrumb).includesText('Secrets Sync', 'Top level breadcrumb renders');
-    assert.dom(cta.button).doesNotExist('CTA does not render');
-    assert.dom(tab('Overview')).hasText('Overview', 'Overview tab renders');
-    assert.dom(tab('Destinations')).hasText('Destinations', 'Destinations tab renders');
-    assert.dom(overview.createDestination).hasText('Create new destination', 'Toolbar action renders');
+    test('it should show create CTA if license has the secrets sync feature', async function (assert) {
```

Comment:
We should stub features here, right? 
```
this.version.features = ['Secrets Sync']
```

Reply:
this value is inherited from the top level, but i'll add it again here for clarity